app/components/Rules.js

```diff
@@ -9,10 +9,10 @@ const Rules = () => {
         <div className="grid">
           <div className="row">
             <div className="col-4">
-              <h2>Setting Up</h2>
+              <h2>Starting a Game</h2>
             </div>
             <div className="col-8">
-              <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis ultricies nisl ex, ac tempus augue rutrum ac. Donec suscipit quam vel aliquet pretium. Phasellus cursus libero in augue pretium, a venenatis justo sagittis. Nam eget leo eget sapien lobortis rhoncus et in lacus. </p>
+              <p>To start a new game, go on <a href="/home">Start Game</a> and enter your username and enter a name for the game. This starts a chat room where you can see who else is playing or who else wants to join the game. In order to start a game, you need at least six players and the person who is initiating the game must click on the Start Game button. Each game will have at least 1 Seer, 1 Doctor, and 2 werewolves, and the rest of the players will be Villagers. Each player should keep their role a secret from other players. </p>
```

Comment:
should we call `the person who is initiating the game` the `game leader`? This is kind of how I phrased it in the Start a new Game component.